src/mpid/ch4/netmod/include/netmod.h

```diff
@@ -17,10 +17,10 @@
 #define MPIDI_MAX_NETMOD_STRING_LEN 64
 
 typedef int (*MPIDI_NM_mpi_init_t) (int rank, int size, int appnum, int *tag_bits,
-                                    MPIR_Comm * init_comm, int *n_vcis_provided);
+                                    MPIR_Comm * init_comm, int *n_vnis_provided);
 typedef int (*MPIDI_NM_mpi_finalize_t) (void);
-typedef MPIDI_vci_resource_t(*MPIDI_NM_vci_get_resource_info_t) (int vci);
-typedef int (*MPIDI_NM_progress_t) (int vci, int blocking);
+typedef MPIDI_vci_resource_t(*MPIDI_NM_vni_get_resource_info_t) (int vni);
+typedef int (*MPIDI_NM_progress_t) (int vni, int blocking);
```

Comment:
The `NM_...` functions are called by upper-layer. Upper layer only knows about logical `vci`. `vni` is netmod entities that are opaque to upper-layer. Therefore, these netmod interface functions should use `vci`. In the implementation of these function, netmod shall map the `vci` into `vni`s.